api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMaker.java

```diff
@@ -0,0 +1,129 @@
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import io.fabric8.kubernetes.api.model.Doneable;
+import io.fabric8.kubernetes.api.model.ObjectMeta;
+import io.fabric8.kubernetes.client.CustomResource;
+import io.strimzi.crdgenerator.annotations.Crd;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import io.sundr.builder.annotations.BuildableReference;
+import io.sundr.builder.annotations.Inline;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyMap;
+import static java.util.Collections.unmodifiableList;
+
+@JsonDeserialize(
+        using = JsonDeserializer.None.class
+)
+@Crd(
+        apiVersion = KafkaMirrorMaker.CRD_API_VERSION,
+        spec = @Crd.Spec(
+                names = @Crd.Spec.Names(
+                        kind = KafkaMirrorMaker.RESOURCE_KIND,
+                        plural = KafkaMirrorMaker.RESOURCE_PLURAL,
+                        shortNames = {KafkaMirrorMaker.SHORT_NAME}
+                ),
+                group = KafkaMirrorMaker.RESOURCE_GROUP,
+                scope = "Namespaced",
+                version = KafkaMirrorMaker.VERSION
+        )
+)
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder",
+        inline = @Inline(type = Doneable.class, prefix = "Doneable", value = "done"),
+        refs = {@BuildableReference(ObjectMeta.class)}
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"apiVersion", "kind", "metadata", "spec"})
+public class KafkaMirrorMaker extends CustomResource {
+
+    private static final long serialVersionUID = 1L;
+
+    public static final String VERSION = "v1alpha1";
+    public static final String RESOURCE_KIND = "KafkaMirrorMaker";
+    public static final String RESOURCE_LIST_KIND = RESOURCE_KIND + "List";
+    public static final String RESOURCE_GROUP = "kafka.strimzi.io";
+    public static final String RESOURCE_PLURAL = "kafkamirrormakers";
+    public static final String RESOURCE_SINGULAR = "kafkamirrormaker";
+    public static final String CRD_API_VERSION = "apiextensions.k8s.io/v1beta1";
+    public static final String CRD_NAME = RESOURCE_PLURAL + "." + RESOURCE_GROUP;
+    public static final String SHORT_NAME = "mm";
+    public static final List<String> RESOURCE_SHORTNAMES = unmodifiableList(asList(SHORT_NAME));
```

Comment:
Speaking `singletonList()` would mean you don't need to `unmodifiableList()`. If you copied this from another CR then please fix those too.